src/app/pages/account/users/user-list/user-list.component.html

```diff
@@ -0,0 +1,120 @@
+<mat-card>
+  <mat-card-content>
+    <table
+      ix-table
+      [dataSource]="dataSource"
+      matSort
+      matSortDisableClear="true"
+      [matSortActive]="defaultSort.active"
+      [matSortDirection]="defaultSort.direction"
+      [multiTemplateDataRows]="true"
+      [fixedLayout]="true"
+    >
+      <ng-container matColumnDef="username">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>{{ 'Username' | translate }}</th>
+        <td mat-cell *matCellDef="let user; dataSource: dataSource">{{ user.username }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="uid">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>{{ 'UID' | translate }}</th>
+        <td mat-cell *matCellDef="let user; dataSource: dataSource">{{ user.uid }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="builtin">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>{{ 'Builtin' | translate }}</th>
+        <td mat-cell *matCellDef="let user; dataSource: dataSource">{{ user.builtin }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="home">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>{{ 'Home' | translate }}</th>
+        <td mat-cell *matCellDef="let user; dataSource: dataSource">{{ user.home }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="full_name">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>{{ 'Full Name' | translate }}</th>
+        <td mat-cell *matCellDef="let user; dataSource: dataSource">{{ user.full_name }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="actions">
+        <th mat-header-cell *matHeaderCellDef></th>
+        <td mat-cell *matCellDef="let user; dataSource: dataSource">
+          <button
+            mat-icon-button
+            type="button"
+            (click)="$event.stopPropagation(); expandRow(user);"
+            [matTooltip]="(expandedRow === user ? 'Collapse' : 'Expand') | translate"
+            aria-label="Expand Row"
+          >
+            <mat-icon>
+              {{ expandedRow === user ? 'expand_less' : 'expand_more' }}
+            </mat-icon>
+          </button>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="expandedDetail">
+        <td mat-cell *matCellDef="let user; dataSource: dataSource" [attr.colspan]="displayedColumns.length">
+          <div class="ix-table-details" [@detailExpand]="user === expandedRow ? 'expanded' : 'collapsed'">
```

Comment:
I guess we could consider extracting everything in this cell to a small component - will make testing easier.